src/Rector/Rector/Namespace_/AppUsesStaticCallToUseStatementRector.php

```diff
@@ -54,21 +57,25 @@ public function getRuleDefinition(): RuleDefinition
      */
     public function getNodeTypes(): array
     {
-        return [FileWithoutNamespace::class, Namespace_::class];
+        return [FileWithoutNamespace::class, Namespace_::class, FunctionLike::class];
```

Comment:
It seems StmtsAwareInterface is better here, to avoid not deleted, eg inside If stmts, I will update.

Reply:
I updated to [make use of StmtsAwareInterface](https://github.com/cakephp/upgrade/pull/236/commits/1ac280bd620c759ac489cbab26ce2b2565d67893)